bin/Execute_report.R

```diff
@@ -4,23 +4,32 @@ library(rmarkdown)
 library(optparse)
 
 option_list = list(
```

Comment:
It would be really nice to group the options here in the same way and order as they are read in the report file